recipe/build.bat

```diff
@@ -52,7 +42,7 @@ if not exist "%MSBUILD_CMD%" (
   /p:Configuration="Release" ^
   /p:Platform="%SLN_PLAT%" ^
   /p:PlatformToolset="%SLN_TOOLSET%" ^
-  /p:WindowsTargetPlatformVersion="%TGT_SDK_VERSION%" ^
+  /t:libusb_dll ^
```

Comment:
There are multiple projects in the solution, targeting just this one ensures only this part is built (the thing we care about).

Reply:
Nice!